tests/roots/test-ext-coverage/python.txt

```diff
@@ -0,0 +1,2 @@
+Undocumented Python objects
+===========================
```

Comment:
No results does not mean exclusion working fine. Please add an example not filtered.

Reply:
Is there reason to store expected result to file?
I prefer to define it inside test script. How about that?

Reply:
Done.

Both the ignored and the non-ignored modules have now the same content, which includes an undocumented class and undocumented methods within an documented class.